Add descriptions to MDX frontmatter (#614)
vercel/turborepo: docs/pages/docs/acknowledgments.mdx

---
title: Acknowledgements and Prior Art

> description: A big thank you to all these developers, build systems, and monorepo tools for the immense support and help which made the development of Turborepo possible.

Reviewer: ```suggestion
description: Thank you to all these developers, build systems, and monorepo tools for their support and assistance.
```

Nit: "A big" - Can be more direct -> Just go straight to "Thank you"
